### DISCUSSION TITLE
RTL loads with incorrect Node ID

### DISCUSSION THREAD

**Discussion by satoshinaka069** [Ride-The-Lightning/RTL#1096] [Troubleshooting Questions] [answered]
Hello, total Noob here with a noob question I'm hoping someone can help answer. I am running Core Lightning and Ride The Lightning (via Umbrel on Raspberry Pi 4).

**PROBLEM:** RTL shows the wrong Alias and NodeID/public key. As such, it's not showing my Lightning balance or any of the channels I've opened using Thunderhub and Lightning Terminal (both work fine). 

![1f6cdaa5edbc0b018887cbd93de37fcbff92566d](https://user-images.githubusercontent.com/113045904/189804024-73073ff5-2081-481d-9247-14fe76ea8fa7.png)

When I check my log file, under Core Lightning it says:

`Server started with public key 032903621c1610f3def7a6359c5356fc157369c9ac74e9c7ef4231a5daab9b8fb3, alias HOPPINGTOTE (color #032903) and lightningd v0.10.2`

But this is not my public key and HOPPINGTOTE is not my Node's alias. Where does Core Lightning get this info from, and how can I change it to my node's actual ID?

I'm also getting the following error under Node Config:

```
Error Code 500
Reading Config Error
```

![c9014c68888d61b8b9869f3ef0bd8e396f47d2e4_2_690x457](https://user-images.githubusercontent.com/113045904/189804042-d8053539-dd94-4281-bfb9-da8274d2bfaf.png)

**Comment by saubyk** [comment, marked as answer]
Hi @satoshinaka069 
So, there are two separate lightning nodes running on your Umbrel:
- LND
- Core Lightning

The node pubkey you see on Umbrel and Thunderhub is your LND node. There are actually two different RTLs available on your Umbrel App Store, RTL (Core Lightning) and RTL. If you install and access RTL you would be able to view your LND node information.

Regarding the Alias for core lightning, that is automatically created by core lightning when it instantiates. You can change it via the core lightning `config` file, but that would require you to access your umbrel via a ssh terminal. If you're not comfortable with operating a command line terminal and do not know your way around a ssh terminal, I would not recommend doing that. Also, I am not sure where or how core lightning `config` can be accessed on umbrel and that question is best addressed to umbrel support.

Regarding the config file error, that again is a question best addressed to the umbrel team, as they need to fix the RTL configuration for core lightning. The core lightning `config` file path has to be correctly configured in `RTL-Config.json` to make this error go away.

Hope this helps. Thanks.

**Reply by satoshinaka069** [reply]
Thank you @saubyk! Boy I feel dumb. I didn’t realize there were two RTL apps. The other version does indeed use the NODE ID that I was looking for.

**Reply by saubyk** [reply]
This is actually a weird way to distribute the app and is confusing for the users, so you shouldn't feel too bad 😄 
Even if Umbrel runs two different lightning nodes, both can be accessed via a single RTL and two different versions of RTL need not be made available on the app store. You can bring this up to Umbrel's attention.
